### DISCUSSION TITLE
How to serialize JSON to disk more efficiently?

### DISCUSSION THREAD

**Discussion by zliucd** [nlohmann/json#3791] [General]
Hi,

I have a number of large JSON objects, say 50MB-1GB,  generated in memory. Writing strings with ```json::dump()``` to disk incurs non-trivial 5-10% overhead. 

This is not an issue of ```nlohmann::json```, but I'm trying to optimize time(not size) for json serialization.  Thanks!

**Comment by nlohmann** [comment]
You may want to check https://json.nlohmann.me/features/binary_formats/ - maybe CBOR can help.
